Make.inc

```diff
@@ -1357,6 +1357,11 @@ endif
 JLIBLDFLAGS += -Wl,-z,noexecstack
 endif
 
+#musl has a really small default stack
```

Comment:
Strictly speaking I think it's only Alpine Linux, but that probably covers vast majority of musl-based systems (and this option should hopefully be good in all other cases)

Reply:
https://git.musl-libc.org/cgit/musl/commit/?id=7b3348a98c139b4 Seems to a musl thing. Well the flag is a linker thing but it seems musl respects it since 2018